priv/repo/migrations/20210602083518_create_tttrecord.exs

```diff
@@ -0,0 +1,14 @@
+defmodule TicTacToeElixir.Repo.Migrations.CreateTttrecords do
+  use Ecto.Migration
+
+  def change do
+    create table(:ttt_records) do
```

Comment:
**thought:** have you considered how you might test this part of the application?